denys/Lesson01-Favorite-Toys/T01.01-Exercise-CreateLayout/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (C) 2016 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<!--TODO (1) Change the ConstraintLayout to a FrameLayout.
+    Note that you don't need to use the fully qualified name for FrameLayout. Replace
+    "android.support.constraint.ConstraintLayout" with "FrameLayout"-->
+<!--TODO (6) Remove the line that declares the id, we don't need it-->
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <!--TODO (2) Remove all attributes with the word constraint in them-->
+    <!--TODO (3) Remove the default text-->
+    <!--TODO (4) Give the TextView 16dp of padding-->
+    <!--TODO (5) Set the text size to 20sp-->
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text=""
+        android:padding="16dp"
+        android:textSize="20sp"
```

Comment:
Note necessary here at all, but I just wanted to mention that you can create these in the `values` directory and then reference to them here, **if** you want different layouts for different screens. For instance the standard padding could go into `res/values/dimens.xml` and another could go into `res/values-w820dp/dimens.xml` (there is an example in your project if you haven't seen).